lib/runtime/interfaces.go

```diff
@@ -4,11 +4,55 @@
 package runtime
 
 import (
+	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/crypto/ed25519"
+	"github.com/ChainSafe/gossamer/lib/keystore"
 	"github.com/ChainSafe/gossamer/lib/transaction"
 	"github.com/ChainSafe/gossamer/lib/trie"
 )
 
+//go:generate mockery --name Instance --structname Instance --case underscore --keeptree
+
+// Instance is the interface a v0.8 runtime instance must implement
+type Instance interface {
```

Comment:
Perhaps keep this `test_helpers.go` for the sake of lower deltas / git blame?
I'm planning on doing a PR sometime to group all interfaces in `interfaces.go` later

Reply:
Oh okay I was thinking good to incrementally move to interfaces.go, but if you're gonna batch do it I can revert and leave as it was

Reply:
Yeah that's the thing. I agree with incremental changes... but on the other hand, it messes up the git blame (like wtf did he move this from a file to another, it's OOS)... And also focusing on doing 1 cleanup in the entire codebase is much more efficient (example change mockery to mockgen), especially given the current tight-ish coupling between packages.